os/arch/arm/src/s5j/s5j_clock.c

```diff
@@ -226,7 +226,7 @@ void s5j_clk_set_rate(unsigned int id, unsigned long rate)
 	case CLK_SPL_SPI1:
 	case CLK_SPL_SPI2:
 	case CLK_SPL_SPI3:
-		clk_spi_set_rate(id, rate);
+		clk_spi_set_rate(id, rate * 2);
```

Comment:
This is difficult to know what it is. Could you kindly add comments for this?

Reply:
@sunghan-chang  sure!
I'll add more information.